### PR TITLE
iOS SDK Release 1.8.3

### PR DESCRIPTION
**Pull request by pavel-kroh-smartlook**
Just new changelog entry and new link to dowload.

### CODE REVIEW

**Review by vaclav-halik-smartlook — CHANGES_REQUESTED** [review]
Fix typos please. Thanks.

File: src/sdk-installation/changelogs.mdx [hunk 1]
```diff
@@ -12,6 +12,16 @@ import { usePlatforms } from 'hooks'
 
 This changelog shows all relevant releases for each platform.
 
+<TextBlock visibleOn="ios">{`
+## iOS [1.8.3] - 2021-12-04
+\n
+### Fixed
+- Ocassional crashes caused by \`EXC_BAD_ACCESS KERN_INVALID_ADDRESS\`.
```
[the remark this answers is not in the document]
Comment (pavel-kroh-smartlook) [reply src/sdk-installation/changelogs.mdx]: Fixed.

File: src/sdk-installation/changelogs.mdx [hunk 2]
```diff
@@ -12,6 +12,16 @@ import { usePlatforms } from 'hooks'
...
+- Safer handling of video pixel buffer pool to avoid \`Fatal Exception: NSInternalInconsistencyException\` crashes with uncertain true cause.
```
[the remark this answers is not in the document]
Comment (pavel-kroh-smartlook) [reply src/sdk-installation/changelogs.mdx]: Reworded.

**Review by vaclav-halik-smartlook — COMMENTED** [review]
OK

**Review by vaclav-halik-smartlook — APPROVED** [review]

**Pull request merged**